hack/jenkins/release_build_and_upload.sh

```diff
@@ -40,6 +40,9 @@ cat Makefile | grep "VERSION_BUILD ?=" | grep $VERSION_BUILD
 BUILD_IN_DOCKER=y make -j 16 all out/minikube-installer.exe out/minikube_${DEB_VERSION}.deb out/minikube-${RPM_VERSION}.rpm
```

Comment:
While we're here - can this rule be made more specific than `all`?

`all` can unexpectedly when multiple rules share the same output name, such as driver rules which can run in docker, or outside of docker. I suspect this may be the root cause of #4555 - but even if it isn't, it will make the result more explicit.